### PR TITLE
Fix dictionnary for multicompany

### PR DESCRIPTION
**Pull request by ptibogxiv**
several fixesfor better management of dictionnary with multicompany

### CODE REVIEW

**Comment by hregis** [comment]
@ptibogxiv
Why would you share the dictionary?
À dictionary is with entity 1 for all entities or current entity when each entity use his own dictionary. You see?

**Comment by ptibogxiv** [comment]
@hregis actually with dolibarr v14 v15, share dictionnary is no fonctionnal with other entity than 1 for social networks

share  or not share dictionnary can be an option (i need both cases)

**Comment by hregis** [comment]
@ptibogxiv ok thanks

**Comment by ptibogxiv** [comment]
/admin/company_socialnetworks.php not functionnal here or  on thridparty/member page

**Comment by hregis** [comment]
@ptibogxiv 
this is fixed in Multicompany 14.0.1
reload install 13 to 14 and Multicompany upgrade all the dictionary for each entity

File: htdocs/admin/dict.php [hunk 1]
```diff
@@ -207,8 +207,8 @@
...
 $tabsql[10] = "SELECT t.rowid, t.code, t.taux, t.localtax1_type, t.localtax1, t.localtax2_type, t.localtax2, c.label as country, c.code as country_code, t.fk_pays as country_id, t.recuperableonly, t.note, t.active, t.accountancy_code_sell, t.accountancy_code_buy FROM ".MAIN_DB_PREFIX."c_tva as t, ".MAIN_DB_PREFIX."c_country as c WHERE t.fk_pays=c.rowid";
 $tabsql[11] = "SELECT t.rowid as rowid, t.element, t.source, t.code, t.libelle, t.position, t.active FROM ".MAIN_DB_PREFIX."c_type_contact AS t";
-$tabsql[12] = "SELECT c.rowid as rowid, c.code, c.libelle, c.libelle_facture, c.nbjour, c.type_cdr, c.decalage, c.active, c.sortorder, c.entity FROM ".MAIN_DB_PREFIX."c_payment_term AS c WHERE c.entity = ".getEntity($tabname[12]);
-$tabsql[13] = "SELECT c.id    as rowid, c.code, c.libelle, c.type, c.active, c.entity FROM ".MAIN_DB_PREFIX."c_paiement AS c WHERE c.entity = ".getEntity($tabname[13]);
+$tabsql[12] = "SELECT c.rowid as rowid, c.code, c.libelle, c.libelle_facture, c.nbjour, c.type_cdr, c.decalage, c.active, c.sortorder, c.entity FROM ".MAIN_DB_PREFIX."c_payment_term AS c WHERE c.entity IN (".getEntity($tabname[12]).")";
```
Comment (eldy) [inline htdocs/admin/dict.php:210]: When you are on entity X, you should see only dic values for entity X.

**Pull request closed without merging**